index.html

```diff
@@ -56,7 +56,7 @@ <h2>let’s connect</h2>
    
   </section>
   <section class="tonic">
-    <img class="imagen-contenedor" src="assets/tonic.png" alt="">
+    <img class="imagen-contenedor" src="assets/tonic-web.svg" alt="">
     <section class="tonicc">
```

Comment:
- [ ] **[OPTIONAL]** Great job addressing the responsiveness issue, **as pointed out by the previous reviewer!** I have a small, **optional suggestion that could enhance the HTML semantics.** You might want to consider grouping all your cards within a `section` instead of using a separate section for each card. Using a `div` or **article** tag for each card could also be a good alternative. **Keep up the good work! 🎯**